applications/nrf_desktop/src/util/hwid.h

```diff
@@ -9,6 +9,14 @@
 
 #define HWID_LEN 8
 
+#ifdef __cplusplus
```

Comment:
Consider also wrapping `#define HWID_LEN 8` (for consistency with other headers in the NCS).
Can we apply the same also to other headers in `src/util/` directory?

Reply:
It might be we will need to include the dfu lock too at some point so let's do it.